tests/OSPSuite.InstallationValidator.Tests/Presentation/MainPresenterSpecs.cs

```diff
@@ -35,49 +35,22 @@ protected override void Context()
 
          _applicationConfiguration = A.Fake<IApplicationConfiguration>();
          A.CallTo(() => _applicationConfiguration.IssueTrackerUrl).Returns(Constants.ISSUE_TRACKER_URL);
-         sut = new MainPresenter(_mainView, _dialogCreator, _batchStarterTask, _batchComparisonTask, _applicationConfiguration);
+         _validationReportingTask = A.Fake<IValidationReportingTask>();
+         sut = new MainPresenter(_mainView, _dialogCreator, _batchStarterTask, _batchComparisonTask, _applicationConfiguration, _validationReportingTask);
       }
    }
 
-   public class When_cancaling_a_validation_is_declined : concern_for_MainPresenter
```

Comment:
I removed those tests because there isn't a way for me to make the presenter think that the validation operation is started anymore. That's because we had an issue where hitting the stop or close always prompted the user to confirm cancel. Now if the validation isn't running, there's no prompt and I can't force it.

That's all.